web-react-ts/src/pages/services/ServicesMenu.tsx

```diff
@@ -103,15 +104,17 @@ const Services = () => {
                 />
               </>
             )}
-          <MenuButton
-            iconComponent={CashCoin}
-            title="Midweek Banking"
-            color="banking"
-            onClick={() =>
-              navigate(`/services/${churchType.toLowerCase()}/midweek-banking`)
-            }
-            noCaption
-          />
+          {church.stream_name === 'anagkazo' && (
+            <RoleView roles={permitLeaderAdmin('Stream')}>
```

Comment:
Here should just permit the admin. Not the lease and admin.

Reply:
I have changed that